src/commcare_cloud/ansible/group_vars/all.yml

```diff
@@ -76,6 +77,7 @@ s3_blob_db_url: "http://{{ groups['proxy'][0] }}:{{ temp_riakcs_proxy_port if 'B
 # - remove `localsettings.TEMP_RIAKCS_PROXY: True` from environments/$ENV/public.yml
 # - deploy proxy (new=8080)
 old_s3_blob_db_url: "http://{{ groups['proxy'][0] }}:{{ riakcs_proxy_port }}"
+old_s3_blob_db_s3_bucket: 'blobdb'
```

Comment:
because now we're in the realm of these being Amazon S3 buckets and S3 buckets are global namespace within all of Amazon S3 customers. So calling our bucket "blobdb" doesn't make sense for S3. We're not changing it for the old one (riakcs) for staging, but the new one (S3) has a custom bucket name:
```
+s3_blob_db_s3_bucket: 'dimagi-commcare-staging-blobdb'
```

Reply:
:+1: I missed that this was getting overridden in the env vars